bundles/org.openhab.core.voice/src/main/java/org/openhab/core/voice/VoiceManager.java

```diff
@@ -106,11 +106,11 @@ public interface VoiceManager {
      * Interprets the passed string using a particular HLI service and the default locale.
      *
      * @param text The text to interpret
-     * @param hliId The id of the HLI service to use or null
+     * @param hliIdList Comma separated list of HLI service ids to use or null
      * @throws InterpretationException
      * @return a human language response
      */
-    String interpret(String text, @Nullable String hliId) throws InterpretationException;
+    String interpret(String text, @Nullable String hliIdList) throws InterpretationException;
```

Comment:
See above, don't remove, add to API if possible. Use `List<String>` for multiple interpreters.

Reply:
As @lolodomo said, we are not really breaking the api here. 
After taking a look all the consumers of this method requires allow passing many interpreters, so if I create another one which accepts List<string> all the consumer will use that instead with the inconvenience of handling the "split by comma" operation, so I think this is want we want. Reopen this if I'm wrong please.